index.xml

```diff
@@ -4040,8 +4042,6 @@ dictionary <i>Derived</i> : <em>Base</em> {
             <li>the type is a dictionary that <a class='dfnref' href='#dfn-dictionary-inherit'>inherits</a> from <var>D</var></li>
             <li>the type is a <a class='dfnref' href='#dfn-nullable'>nullable type</a>
             whose <a class='dfnref' href='#dfn-inner-type'>inner type</a> includes <var>D</var></li>
-            <li>the type is a <a href='#idl-sequence'>sequence type</a> or <a href='#idl-array'>array type</a>
```

Comment:
Why can the sequence bits here go away?  Shouldn't they stay but `FrozenArray` get added?